pkg/domain/infra/abi/images.go

```diff
@@ -294,6 +294,10 @@ func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entiti
 	return &entities.ImagePullReport{Images: foundIDs}, nil
 }
 
+func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entities.ImagePullOptions) (*entities.ImagePullReport, error) {
+	return pull(ir.Libpod.ImageRuntime(), ctx, rawImage, options)
```

Comment:
This change seems out of context of the bug fix.

Reply:
The basic idea was to use one consistent way of pulling an image.  I found bugs in the runlabel pull code, that were different then podman pull, so rather then cut and paste in the fixes, I created a new shared function that both could use.